packages/lesswrong/components/notifications/NotifyMeButton.tsx

```diff
@@ -97,23 +108,28 @@ const NotifyMeButton = ({
     {message}
   </>
 
-  const {MenuItem} = Components;
-  const maybeMenuItemButton = asMenuItem ?
+  // Determine if this component should look like a link, menu item, or button
+  let maybeMenuItemButton = asMenuItem ?
     <MenuItem onClick={onSubscribe}>
       <a className={classNames(classes.root, className)}>
         {button}
       </a>
-    </MenuItem> : asButton ?
-    <button onClick={onSubscribe} className={classNames(className, classes.root)}>
-      {button}
-    </button> :
-    <a onClick={onSubscribe} className={classNames(className, classes.root)}>
+    </MenuItem> : <a onClick={onSubscribe} className={classNames(className, classes.root)}>
       {button}
     </a>
+  if (asButton) {
+    maybeMenuItemButton = isFriendlyUI ? (
+      <EAButton style="grey" onClick={onSubscribe}>{button}</EAButton>
+    ) : (
+      <button onClick={onSubscribe} className={classNames(className, classes.root)}>
+        {button}
+      </button>
```

Comment:
I think the `asButton` version was only used in the post author card anyway, which is no longer on any site